database.sql

```diff
@@ -333,33 +333,14 @@ CREATE TABLE `event` (
   id INT PRIMARY KEY AUTO_INCREMENT NOT NULL ,
   imgPath TEXT ,
   title VARCHAR(255) NOT NULL,
+  course VARCHAR(255) NOT NULL,
+  raceDate DATE NOT NULL,
   paragraph TEXT NOT NULL
 );
 
 INSERT INTO `event` (title, paragraph) VALUES ("Compétition rallye Orleans", "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum id sodales tellus. Pellentesque in lorem vitae risus fermentum rutrum. 
```

Comment:
si tu rajoutes 2 champs "not null" et que tu fais un insert sans les spécifier, ça doit planter... tu as relancé un migration.php ?
mais au delà de ça, autant je comprend l'ajout d'un champ date, autant le champ course j'ai du mal à voir la diff avec le title

Reply:
euh le title c pour le nom de la course et le course c'est le type de course (auto moto.....)